pom.xml

```
@@ -42,7 +42,7 @@
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
 
     <!-- jenkins versions -->
-    <jenkins.version>2.426.3</jenkins.version>
+    <jenkins.version>2.472</jenkins.version>
```

Since you did the work to remove the Jetty dependency from the test, there is no need to increase the minimum Jenkins version.  The minimum Jenkins version can stay the same as it was because your test improvement works with Java 11, Java 17, and Java 21 and it works with all the Jenkins versions supported by this plugin.

```suggestion
    <jenkins.version>2.426.3</jenkins.version>
```